WcaOnRails/.rubocop.yml

```diff
@@ -54,3 +54,9 @@ Style/SignalException:
 Style/ClassVars:
   Exclude:
     - app/models/country.rb
+
+Style/IfInsideElse:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
```

Comment:
We're still breaking the `Style/IfInsideElse` rule [here](https://github.com/jfly/worldcubeassociation.org/blob/56182b003d0bd370e19db334d4513d4406fe58df/WcaOnRails/app/models/result.rb#L50-L54) and [here](https://github.com/jfly/worldcubeassociation.org/blob/56182b003d0bd370e19db334d4513d4406fe58df/WcaOnRails/app/models/result.rb#L87-L95), but I like breaking it to achieve parallelism.

I have removed the `Lint/NonLocalExitFromIterator`.